app/components/profile_picture/image.tsx

```diff
@@ -44,10 +44,10 @@ const Image = ({author, forwardRef, iconSize, size, source, url}: Props) => {
     const lastPictureUpdateAt = author ? getLastPictureUpdate(author) : 0;
     const fIStyle = useMemo(() => ({
         borderRadius: size / 2,
-        backgroundColor: theme.centerChannelBg,
+        backgroundColor: changeOpacity(theme.centerChannelColor, 0.12),
```

Comment:
@matthewbirtch per request

> by just adding a 12% opacity center channel color fill

Only helps if the image is transparent.  If the image is the same color as centerChannelColor, then you'd have the same problem.